torch/nn/quantized/modules/batchnorm.py

```diff
@@ -21,11 +21,9 @@ def _get_name(self):
 
     @classmethod
     def from_float(cls, mod):
+        activation_post_process = mod.activation_post_process
```

Comment:
any chance you could expand on the reasoning for this change?

Reply:
sure, previous code is relying on the fact the we attach activation_post_process to ReLU inside the sequential, but this PR removes activation_post_process for ReLU, so we changed the insert observer logic to add activation_post_process to sequential directly so we don't need to get activation_post_process from relu any more
```
activation_post_process = mod[1].activation_post_process	
```
this is getting activation_post_proceess from relu.

Reply:
makes sense, thanks!  maybe we can add a quick comment to the code?

Reply:
will add a comment to the place we attach activation_post_process for _FusedModule (in quantization/quantize.py)